zerver/tests/test_custom_profile_data.py

```diff
@@ -472,6 +535,20 @@ def test_update(self) -> None:
         )
         self.assert_json_error(result, "Label cannot be blank.")
 
+        self.assertEqual(CustomProfileField.objects.count(), self.original_count)
+        result = self.client_patch(
+            f"/json/realm/profile_fields/{field.id}",
+            info={"name": "New phone number"},
```

Comment:
We already test changing name below. So, can we remove this?

Reply:
Sorry, I seem to have messed up a merge conflict. Removed these lines.